language/fr/acp_ajax_shoutbox.php

```diff
@@ -10,8 +10,8 @@
  */
 
 /**
- * DO NOT CHANGE
- */
+* DO NOT CHANGE
+*/
 if (!defined('IN_PHPBB'))
```

Comment:
Please undo these whitespace changes (This applies for the other files as well).